maven/bnd-export-maven-plugin/README.md

```diff
@@ -7,24 +7,38 @@ The `bnd-export-maven-plugin` is a bnd based plugin to export bndrun files.
 Point the plugin to a bndrun file in the same project. It will export a runnable jar.
 
 ```
-            <plugin>
-                <groupId>biz.aQute.bnd</groupId>
-                <artifactId>bnd-export-maven-plugin</artifactId>
-                <version>${bnd.version}</version>
-                <configuration>
-                    <failOnChanges>false</failOnChanges>
-                    <bndruns>
-                        <bndrun>mylaunch.bndrun</bndrun>
-                    </bndruns>
-                </configuration>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>export</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
+    <plugin>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-export-maven-plugin</artifactId>
+        <version>${bnd.version}</version>
+        <configuration>
+            <failOnChanges>false</failOnChanges>
+            <bndruns>
+                <bndrun>mylaunch.bndrun</bndrun>
+            </bndruns>
+        </configuration>
+        <executions>
+            <execution>
+                <goals>
+                    <goal>export</goal>
+                </goals>
+            </execution>
+        </executions>
+    </plugin>
+```
+
+Here's an example setting the `bundles` used for resolution.
+
+```
+    ...
+    <configuration>
+        ...
+        <bundles>
+            <bundle>bundles/org.apache.felix.eventadmin-1.4.8.jar</bundle>
```

Comment:
should I just add this to the documentation? The paths are relative to the project dir as it is. This is handled by maven internals as it is.